fusillade-api.yml

```diff
@@ -30,10 +33,40 @@ paths:
           description: OK
         default:
           description: Unexpected error
-  /v1/oauth/authorize:
+  /.well-known/openid-configuration:
+    get:
+      summary: This endpoint is part of OIDC
+      tags:
+        - oauth
+      operationId: fusillade.api.oauth.serve_openid_config
+      responses:
+        200:
+          description: Returns the supported openid endpoints
+  /.well-known/jwks.json:
+    get:
+      summary: This endpoint is part of OIDC
+      tags:
+        - oauth
+      operationId: fusillade.api.oauth.serve_jwks_json
+      responses:
+        200:
+          description: Returns the well know jwks
+  /cb:
+    get:
+      summary: An internal endpoint that should not be used.
+      tags:
+        - oauth
+      operationId: fusillade.api.oauth.cb
+      responses:
+        200:
+          description: return a authorization token
+  /oauth/authorize:
     get:
       summary: This endpoint is part of OIDC and is used to redirect to an openid provider
         see https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest
```

Comment:
Does this line wrap work appropriately? There should be a semicolon after "provider"

Reply:
fixed